src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -11,17 +12,23 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class ConsignmentRepository(db: Database) {
 
+  implicit class ZonedDateTimeUtils(value: ZonedDateTime) {
+    def toTimestamp: Timestamp = {
+      Timestamp.valueOf(value.toLocalDateTime)
+    }
```

Comment:
You've already got a `ZonedDateTimeUtils` implicit class in `FieldTypes` Is it worth merging this method into that?

Reply:
Fair point, was thinking about this. I think it actually might be worth creating a new class `TimeUtils` and adding the implicit classes to this. Wasn't too comfortable putting this in the FieldTypes as these implicits can be used across the whole project, not just the graphql types. What do you reckon?

Reply:
Yeah that makes sense. Once my changes for the metadata go in there'll be LocalDateTime stuff so I can put them in `TimeUtils` as well.